src/hooks/useProductList.js

```diff
@@ -1,10 +1,15 @@
+import Product1 from '../assets/product-imgs/Product1.svg'
+import Product2 from '../assets/product-imgs/Product2.svg'
+import Product3 from '../assets/product-imgs/Product3.svg'
+import Product4 from '../assets/product-imgs/Product4.svg'
```

Comment:
I know this is fake data, but we probably should name them better, just that when we read the code, we can imagine the layout and the visual components. 

Also, files should be snake-case. `product4` is fine.